apps/user-core-migration/migrate_institution_description.py

```diff
@@ -16,12 +16,55 @@
 USERS_DB = 'selcUser'
 USER_INSTITUTION_COLLECTION = 'userInstitutions'
 
+ONBOARDING_DB = 'selcOnboarding'
+ONBOARDINGS_COLLECTION = 'onboardings'
+
 BATCH_SIZE = 100
 START_PAGE = 0
-INSTITUTION_ID = None if len(sys.argv) < 2 else sys.argv[1]
+TAX_CODE = None if len(sys.argv) < 2 else sys.argv[1]
+
+def migrate_institution_description_from_onboarding_to_core(client):
+    if TAX_CODE is None:
+        onboardings_size_cursor = client[ONBOARDING_DB][ONBOARDINGS_COLLECTION].aggregate(count_institutions_from_onboardings())
+        onboardings_size = next(onboardings_size_cursor)['count']
+        print("Onboarding's institutions size: " + str(onboardings_size))
+        pages = math.ceil(onboardings_size / BATCH_SIZE)
+
+        for page in range(START_PAGE, pages):
+            print("Start page " + str(page + 1) + "/" + str(pages))
+
+            onboarding_pages = client[ONBOARDING_DB][ONBOARDINGS_COLLECTION].aggregate(
+                get_institutions_from_onboardings(page, BATCH_SIZE)
+            )
+
+            for institution in onboarding_pages:
+                institution_desc = institution.get('description')
+                institution_tax_code = institution.get('_id')
```

Comment:
should it be institution_id?

Reply:
_id field is returned from groupBy operation and query group by taxCode in onboarding collection.